statistics/handle/cache/statscacheinner.go

```diff
@@ -105,10 +122,30 @@ func (sc *StatsCache) PutFromInternal(id int64, t *statistics.Table) {
 	sc.put(id, t, false)
 }
 
+func (sc *StatsCache) putCache(id int64, t *statistics.Table, moveLRUFront bool) bool {
+	ok := sc.c.Put(id, t, moveLRUFront)
+	if ok {
+		return ok
+	}
+	// retry three times and sleep exponentially. 20ms, 40ms and 80ms.
+	for i := 0; i < 3; i++ {
+		time.Sleep(time.Duration(1<<uint(i)) * 10 * time.Millisecond)
+		ok = sc.c.Put(id, t, moveLRUFront)
+		if ok {
+			return ok
+		}
+	}
```

Comment:
Please don't do that... this retry seems very dangerous, if the `StatsHandler` call this, in the worst case, `StatsHandler` will be blocked for `140ms`.
I suggest that we just log a warning here and do nothing about retry.